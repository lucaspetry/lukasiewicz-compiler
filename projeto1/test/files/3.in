int bas_ = 32
A = 2
