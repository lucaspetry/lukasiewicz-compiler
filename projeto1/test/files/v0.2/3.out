var float: abba, ikea
= abba 12.0
= ikea + * 13.21 1.7 4.2
var bool: svenska
= svenska <= abba ikea

